### ISSUE TITLE
Label of the y-axis of the weighted histograms 

### ISSUE THREAD

**Issue by mar-mueller** [wdecoster/NanoPlot#311]
Hello, 

I have a question regarding the label of the y-axis of the weighted histograms. My weighted histrograms all have "Number of reads" on the y-axis what doesn't really make sense to me because the "number of reads" in the histograms would exceed the total number of reads that I obtained. In older histograms that were posted the y-axis of the weighted histograms were always labeled with "number of bases". Why are they now labeled with number of reads? (I added an example)

Thanks!
![WeightedHistogramReadlength](https://user-images.githubusercontent.com/114082355/192774968-a3b91d58-7f3c-4b38-bb98-9205c6d0bfd6.png)

**Comment by wdecoster** [comment]
You are right, that label is incorrect and should still be `number of bases`. I will correct this.
